atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -165,7 +161,7 @@ def update_config_from_dockerfile(self, config):
 
         :param config: ConfigParser object
         """
-        labels = Labels(df_parser(self.workflow.df_path).labels)
+        labels = Labels(self.workflow.build_dir.any_platform.dockerfile.labels)
```

Comment:
Hmm, sometimes when parsing labels we inject the parent environment and sometimes we don't.

The content_manifest plugin, for example, uses `dockerfile_with_parent_env`, this just uses  `dockerfile`. And that *is* consistent with how it was done before - this used df_parser without the workflow param, so no parent env :thinking: 

It doesn't seem to matter that much, I'll just file something as potential future improvement

Reply:
ok